### PR TITLE
Removes hybrid friend finder.

### LINKED ISSUE #3572: Remove Hybrid Friend Finder
Title has it. The hybrid feature is really too broken to justify keeping.  We'll reintroduce it in #1200.
Refs: #3408, #1335, #3331, and #540.

### PR DESCRIPTION
**Pull request by aerych**
Closes #3572

### CODE REVIEW

**Comment by aerych** [comment]
cc @SergioEstevao are you game for a quick review?

**Comment by koke** [comment]
post-31310-thats-wonderful-gif-andre-the-54ak

You get two GIFs, that's how much I like this

tumblr_mxx10xvlup1t8quc8o1_500

File: WordPress/Classes/ViewRelated/Reader/WPFriendFinderViewController.m [hunk 1]
```diff
@@ -1,275 +0,0 @@
-#import <AddressBook/AddressBook.h>
-#import <Accounts/Accounts.h>
-#import <Social/Social.h>
```
Comment (koke) [inline WordPress/Classes/ViewRelated/Reader/WPFriendFinderViewController.m]: Looks like this was the only file using those three frameworks, maybe we can stop linking them.

File: WordPress/Classes/ViewRelated/Reader/WPFriendFinderViewController.m [hunk 2]
```diff
@@ -1,275 +0,0 @@
...
-
-@end
-
-@implementation WPFriendFinderViewController
-
-- (void)viewDidLoad
-{
-    [super viewDidLoad];
-
-    [self loadURL:WPMobileReaderFFURL];
```
Comment (koke) [inline WordPress/Classes/ViewRelated/Reader/WPFriendFinderViewController.m]: That URL is still in the `Constants` file

**Comment by koke** [comment]
Other that those cleanup comments, looks good to me :+1: 
Happy to se it go :smile:

File: WordPress/Classes/ViewRelated/Reader/WPFriendFinderViewController.m (hunk 2 above)
Comment (aerych) [inline WordPress/Classes/ViewRelated/Reader/WPFriendFinderViewController.m]: good catch

**Comment by aerych** [comment]
Thanks @koke :)  I tested these changes but would you mind giving it another round for paranoia sake?

**Comment by koke** [comment]
I think it's good. Not much to test on removed code ;)

**Comment by aerych** [comment]
> I think it's good. Not much to test on removed code ;)

True! But paranoia FTW :)   Thanks @koke!

**Pull request merged**